### PR TITLE
Configure Release Drafter GitHub action

### PR DESCRIPTION
**Pull request by swissspidy**
See #319.

### CODE REVIEW

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - title: '🚀 Features'
+    labels:
+      - 'enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'bug'
+  - title: '🧰 Maintenance'
+    labels:
+      - 'dependencies'
+change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
```
Comment (swissspidy) [inline .github/release-drafter.yml]:
```suggestion
change-template: '- $TITLE (#$NUMBER)'
```

File: .github/release-drafter.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name-template: '$NEXT_PATCH_VERSION 🌈'
```
Comment (swissspidy) [inline .github/release-drafter.yml]:
```suggestion
name-template: '$NEXT_PATCH_VERSION'
```

**Comment by swissspidy** [comment]
@pbakaus Would this help for your weekly reviews? It basically keeps an automated changelog based on all the merged PRs/issues.

**Comment by pbakaus** [comment]
yes!

**Comment by swissspidy** [comment]
Cool, let's give it a shot then 👍

**Pull request merged**